### PR TITLE
feat: Add story for Alert component

### PR DESCRIPTION
**Pull request by Ajalle**

### CODE REVIEW

**Comment by NoopDog** [comment]
@Ajalle when this is ready can you PR to @frano-m, please? Thanks! D

**Comment by Ajalle** [comment]
image

File: packages/data-explorer-ui/src/components/common/Alert/alert.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    },
+    severity: {
+      description: "Top left predefined icon. Combine with color.",
+    },
+    title: {
+      control: "text",
+      description: "Highlighted text",
+    },
+    variant: {
+      description: "[deprecated] Type of the alert",
```
Comment (frano-m) [inline packages/data-explorer-ui/src/components/common/Alert/alert.stories.tsx:26]: Hi @Ajalle - would you mind explaining the use of the word `[deprecated]` here in this context? 🤔

**Review by frano-m — APPROVED** [review]
Awesome thank you @Ajalle

**Pull request merged**